CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   the node's and edge's dictionaries, respectively (e.g., 'classes', 'positions', etc.).
 * Removed `Tree`'s method `add_child`, because it is redundant with `add_children` called with an
   argument of length 1.
+* Contributed initial build of R package.
```

Comment:
This could probably go in "Added"!

Reply:
Right, that makes more sense. Will migrate this bit.

Reply:
fixed in c969f1e